apps/macro-sandbox/functions/python/handler.py

```
@@ -3,9 +3,19 @@
 import gzip
 import subprocess
 import os
+import sys
 import shutil
 import tempfile
 
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), "guards"))
+from guard import guard_batch, partition_items, merge_results  # noqa: E402
+
+
+def _guard_mode():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_guard_mode`

Add return type annotation: `str`

(ANN202)